### ISSUE TITLE
[jquery-selfpaced] Create follow_up.md

### ISSUE THREAD

**Issue by karlalopez** [gdisf/teaching-materials#423]
1. Create `follow_up.md` in the lesson folder
2. Add content:

```
Thanks for attending the workshop, and thank you to our TAs [TODO: TA names] for being such great helpers. We’d love your feedback on the workshop in this survey. [TODO: Add link to survey]

The slides are available at:
http://www.teaching-materials.org/jquery-selfpaced/

jQuery is deceptive in its simplicity - it’s a small amount of code to write, but there’s a lot to understand to really be able to use it well. So don’t worry if you’re feeling a bit overwhelmed, you learned a lot of syntax *and* concepts today, so it’s totally expected.

As usual, the important thing now is to keep practicing what you’ve learnt, at home or at the study groups.

After you finish the Khan Academy course, you can also continue learning jQuery via these online resources:

- [jQFundamentals](http://jqfundamentals.com/)
- [jQuery](http://www.codecademy.com/tracks/jquery)
- [15 jQuery Resources](http://net.tutsplus.com/tutorials/javascript-ajax/15-resources-to-get-you-started-with-jquery-from-scratch/)
- [AppendTo](http://learn.appendto.com/lesson/selectors-101)

The upcoming AJAX workshop [TODO: add link to meetup] will show you how to pull data into your site, using either native DOM API or jQuery.


Stay in touch through [Facebook](https://www.facebook.com/gdisf/), [Twitter](https://twitter.com/gdisf), and [Slack](http://gdisf.slack.com). (Not a member? [Sign up for GDI SF Slack here](http://gdisf-slack.herokuapp.com)). Hope to see you at future workshops!
```

3. On /index.html, look for the **JS301: jQuery (Online, Self-paced)** row and add the link to `follow_up.md`:
```
| <a href="https://github.com/gdisf/teaching-materials/blob/jquery-selfpaced/follow_up.md">Follow Up Email</a>
```

4. Test the links

**Comment by Slamare** [comment]
@slamare I will work on this

**Comment by verythorough** [comment]
Closed by #501. Thank you, @Slamare!
